pram

```diff
@@ -218,6 +224,19 @@ main() {
 			continue
 		fi
 
+		if ${interactive} ; then
+			local answer
+			read -p "Do you want to merge this? (y/n) " answer
+			case ${answer} in
+				y)
+					:;
+					;;
+				*)
+					echo "Merge has been skipped!"
+					return 0
```

Comment:
You are omitting remaining parameters here. If you really want to do that, please add extra `/q` for quit.

Reply:
I do not really understand what you mean here. Care to explain a bit more?

Reply:
Would changing it to `continue` be sufficient?

Reply:
Yes, probably. Just make sure it doesn't skip something necessary below.

I meant that you can pass multiple PRs to it, and you don't want 'n' to mean 'skip all remaining PRs'. I suppose we'd want two options: 'n' to skip current, 'q' to quit (= skip all).

Reply:
How about this?